src/modules/DASDController.rb

```diff
@@ -137,7 +147,17 @@ def Write
             to_reactivate << device
           end
           if format
-            to_format << (device["dev_name"] || GetDeviceName(channel))
+            if can_be_formatted?(device)
+              to_format << (device["dev_name"] || GetDeviceName(channel))
+            else
+              Report.Error(
+                # TRANSLATORS %s is device name
+                format(
+                  _("Cannot format device '%s'. Only ECKD can be formatted."),
```

Comment:
Same comment than above about the string... which makes me realize there is some code duplication. Nothing too bad taking into account this is yast2-s390. :-)

Reply:
With "same comment than above" I meant using "Only ECKD **disks**"

Reply:
now I get it, I think you mean that traslators :)